OxfordBodleian/Juel-Jensen/BDLaethg30.xml

```diff
@@ -225,12 +234,15 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                     </physDesc>
 
                     <history>
+                        <!--
+                            moved to extras, but might be summarized in origin:
+                            The name of the original owner, <persName ref="PRS13686WaldaTH"/>, is written in various supplication formulas. It is fully readable on <locus target="#31v"/>
+                        and partially legible on <locus from="23r" to="23v"/> (where only Walda is legible) and on <locus target="#28r"/> (where only Walda Takla is legible).-->
                         <origin>
                             <origDate notBefore="1800" notAfter="1950" evidence="lettering"/>
                         </origin>
-                        <provenance>Unknown provenance. The name of the original owner, <persName ref="PRS13686WaldaTH"/>, is written in various supplication formulas. It is fully readable on <locus target="#31v"/>
-                            and partially legible on <locus from="23r" to="23v"/> (where only Walda is legible) and on <locus target="#28r"/> (where only Walda Takla is legible).
-                            The manuscript came to <persName ref="PRS5782JuelJen"/> under unknown circustamnces.</provenance>
+                        <provenance>Unknown provenance. 
```

Comment:
Has this been recorded systematically for all the manuscripts?

Reply:
no, this will be part of the history revision